### ISSUE TITLE
(바보짓 기록) AttributeError at DataLoader

### ISSUE THREAD

**Issue by wondrive** [wondrive/error_i_met#10]
# Error Message
```
-> train_loader = DataLoader(train_dataset, batch_size=4, shuffle=True)
AttributeError
```
- AttributeError 외에 별다른 에러 메세지는 안 떴다. 그래서 더욱 원인을 알 수 없었다.
- 삽질한 것: shuffle=False로 바꾸면 에러는 안 뜨길래 (대신 다른 문제는 생김.. next(iter~로 불러오는 게 안 됨) 그게 문제인줄 알았음

# Solution
`train_loader = DataLoader(dataset=train_dataset, batch_size=4, shuffle=True)`
`dataset=` 라고 attrbute name 명시해주니까 해결됨
